code/modules/events/holiday/vday.dm

```diff
@@ -75,15 +76,16 @@
 	               "Roses are red, text is green, I love you more than cleanbots clean.",
 	               "If you were a carp I'd fi-lay you." )
 
-/obj/item/weapon/valentine/attackby(obj/item/weapon/W, mob/user, params)
+/obj/item/weapon/hand/valentine/attackby(obj/item/weapon/W, mob/user, params)
+	..()
 	if(istype(W, /obj/item/weapon/pen) || istype(W, /obj/item/toy/crayon))
 		var/recipient = stripped_input(user, "Who is receiving this valentine?", "To:", null , 20)
 		var/sender = stripped_input(user, "Who is sending this valentine?", "From:", null , 20)
 		if(recipient && sender)
 			name = "valentine - To: [recipient] From: [sender]"
-	..()
+			blank = 0
 
-/obj/item/weapon/valentine/examine(mob/user)
+/obj/item/weapon/hand/valentine/examine(mob/user)
```

Comment:
the hand path is for a hand of cards

see the checklist -- I want to basically make valentines and cards function  the same way I just haven't figured out how best to do it yet